scripts/create_configs.py

```diff
@@ -403,6 +405,37 @@ def define_nondnn_binning(self):
     @staticmethod
     def is_for_limits(spin, mass):
         return False if spin=='' and mass=='' else True
+
+
+def r_factor(y, chan):
+    if chan=="MuMu":
+        return ""
```

Comment:
shouldn't the factor be 1. for "MuMu", instead of the empty string?

Reply:
Thank you for your comments. In the older versions (looking at the March config, etc.), the section bTagRfactor was not present in the config file, while in more recent versions it was included, but with the value commented out. If I understand correctly, in the [combiner](https://github.com/LLRCMS/KLUBAnalysis/blob/736a4aa4fffb8ccc017caa0668d288abce7b4ced/scripts/combineFillerOutputs.py#L134C4-L152C29), in the absence of central=number, it assigns 1

Reply:
I would say better to set it to 1 there, so we're sure:
```
(base) bruno@bruno:~  $ python
>>> import this
The Zen of Python, by Tim Peters
...
Explicit is better than implicit.
...
```

Reply:
ok, updated!